integrations/applications/google-workspace/resources/metadata.json

```diff
@@ -0,0 +1,221 @@
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "domainName"
+							}
+						}
+					]
+				},
+				{
+					"id": "assertion-Consumer-Urls",
+					"aria-label": "Assertion Consumer Urls",
+					"name": "inboundProtocolConfiguration.saml.manualConfiguration.assertionConsumerUrls.[0]",
+					"label": "Assertion Consumer Urls",
+					"type": "text",
+					"required": true,
+					"dataComponentId": "google-workspace-create-wizard-acs",
+					"hidden": true,
+					"handlers": [
+                        {
+                            "name": "dependentProperty",
+                            "type": "submission",
+                            "props": {
+                                "placeholder": "domainName"
+                            }
+                        }
+					]
+				},
+                {
+                    "id": "domainName",
+                    "aria-label": "Domain name",
+                    "name": "domainName",
+                    "label": "Domain name",
+                    "type": "text",
+                    "helperText": "Dedicated google domain obtainded by you.",
+                    "required": true,
+                    "dataComponentId": "google-workspace-create-wizard-domainName",
+                    "hidden": false,
+                    "handlers": [
+                        {
+                            "name": "disableProperty",
+                            "type": "submission"
+                        }
+                    ]
+                },
+				{
+					"id": "access-url",
+					"aria-label": "Access URL",
+					"name": "accessUrl",
+					"label": "Access URL",
+					"type": "text",
+					"required": false,
+					"dataComponentId": "google-workspace-create-wizard-access-url",
+					"hidden": true,
+					"handlers": [
+						{
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "inboundProtocolConfiguration.saml.manualConfiguration.issuer"
+							}
+						},
+						{
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "tenantDomain"
+							}
+						},
+						{
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "serverOrigin"
+							}
+						}
+					]
+				},
+				{
+					"id": "image-url",
+					"aria-label": "Image URL",
+					"name": "imageUrl",
+					"label": "Image URL",
+					"type": "text",
+					"required": false,
+					"dataComponentId": "google-workspace-create-wizard-image-url",
+					"hidden": true,
+					"handlers": [
+						{
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "clientOrigin"
+							}
+						},
+						{
+							"name": "dependentProperty",
+							"type": "submission",
+							"props": {
+								"placeholder": "appBaseNameWithoutTenant"
+							}
+						}
+					]
+				},
+				{
```

Comment:
I think we don't need to keep this application sharable checkbox since it is `hidden: true`?

Reply:
Let's check other places as well